Pod/Common/NSURLRequest+SBTUITestTunnelMatch.m

```diff
@@ -76,7 +76,17 @@ - (BOOL)matches:(SBTRequestMatch *)match
         matchesMethod = [self.HTTPMethod isEqualToString:match.method];
     }
 
-    return matchesURL && matchesQuery && matchesMethod;
+    BOOL matchesBody = YES;
+    if (match.body) {
+        NSString *body = [[NSString alloc] initWithData:self.HTTPBody encoding: NSUTF8StringEncoding];
+        NSRegularExpression *regex = [[NSRegularExpression alloc] initWithPattern:match.body options:nil error:nil];
```

Comment:
What about adding the custom "exclamation mark" convention that we used for path matching to invert the regex? This would allow to match all requests that do not contain something in the body.

Reply:
👍 added in b818d9d